### ISSUE TITLE
email script to ensure nobody locks themselves out of their account

### ISSUE THREAD

**Issue by jeffpaul** [wptide/wptide#115]
### Description

From Derek in Slack:
I just thought of a simple feature. Maybe a plugin already exists. When you save changes to your email the script blocks the change until you click a link sent to the original email. The script would poll the DB and the link sent through email would set the value it's looking for, which would unblock and allow the script to save the changes. Just something to ensure nobody locks themselves out of their accounts. Also adding a cancel button to stop the save.

**Comment by jeffpaul** [comment]
Per [bugscrub](https://wordpress.slack.com/archives/C7TK8FBUJ/p1535487381000100) today, we're moving this to `Future Release` though it would be a nice feature plugin.

**Comment by valendesigns** [comment]
This is way out of scope for the project. Closing it.
